cmd/ocm-backplane/login/login.go

```diff
@@ -183,6 +185,13 @@ func runLogin(cmd *cobra.Command, argv []string) (err error) {
 	}
 
 	logger.Debugf("Using backplane URL: %s\n", bpURL)
+	backplanehost, err := getBackplaneHost(bpURL)
+	if err != nil {
+		logger.Errorf(err.Error())
```

Comment:
If we call error here and won't stop the next, it is a bit weird? Maybe warning is better

Reply:
The commit  e26c5460d1aeb38f4b538aacd41d826975832aa9  changed it to Warning.